src/Toolbox.php

```diff
@@ -3309,4 +3314,13 @@ final public static function getNormalizedItemtype(string $itemtype)
     {
         return strtolower(str_replace('\\', '', $itemtype));
     }
+
+    /**
+     * @param string $message
+     * @return string
+     */
+    public static function cleanPaths(string $message): string
```

Comment:
Is this new method really needed? I do not think we expect it does more one day than just calling `ErrorUtils::cleanPaths()` - which can be done directly as far as I see.

Reply:
I see no reason having now or in the future a `Toolbox::cleanPaths` that does something else than `ErrorUtil::cleanPaths`
I also do not see how it's less error prone; it may even be confusing having 2 distinct method that does exactly the same.

Reply:
did you notice Toolbox::cleanPaths() calls ErrorUtil::cleanPaths() ? There is only one concrete function.
In my experience, it's always a good thing to isolate that kind of things.
It doesn't deserve a long discussion as you said.

Reply:
When we will be able to "inject" dependencies in the constructor, the `ErrorUtil::cleanPaths()` calls may be replaced by `$this->errorUtils->cleanPaths()` calls. The way it is done currently may indeed require less changes. Anyway, this kind of changes is pretty easy to made with IDEs and error would probably be detected by PHPStan or the test suite.

Both way to do are valid to me.